TableController/TableController/Sources/SectionController.swift

```diff
@@ -119,7 +119,7 @@ open class SectionController: NSObject {
     
     open func deleteCellControllers(at indices: [Int], with animation: UITableViewRowAnimation = .automatic) {
         
-        for index in indices {
+        for index in indices.sorted().reversed() {
```

Comment:
If the array of indices was in ascending order, we ran the risk of deleting items and mismatching subsequent indices. By deleting them in descending order, we mitigate this altogether.